server/playground/index.js

```diff
@@ -0,0 +1,28 @@
+const buildSchema = require("data-sync-gql-core");
+const { makeExecutableSchema } = require("graphql-tools");
+
+const { ApolloServer } = require("apollo-server-express");
+const models = require("../models");
+
+function connectDataSources(dataSources) {
```

Comment:
We can refactor top level api to offer something like this:

interface Core {
 buildSchema(schemaName: string): Promise<Schema>
 createGraphQLApi(config: SyncCoreConfig) : Promise<void>  // { app, path: "/graphqldata" }
 connectActiveDataSources(): Promise<void>